dali/pipeline/init.cc

```diff
@@ -12,15 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <signal.h>
 #include "dali/pipeline/init.h"
-
+#include "dali/error_handling.h"
 #include "dali/pipeline/data/backend.h"
 
 namespace dali {
 
+namespace {
+
+void signal_handler(int sig) {
+  std::cerr << "Error: signal " << std::to_string(sig) << ":"
```

Comment:
Isn't `strsignal` that should be used rather than `std::to_string`?

Reply:
done